CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+<<<<<<< HEAD
+=======
+## [0.10.6] - 2022-12-06
+
+### Added
+
+- Added a new entity named `predictionData` to reflect the predictions layout stored in machine learning S3 buckets . 
```

Comment:
This is technical information that doesn't concern the user of this package.
He's not supposed to know or care that the predictions layout are stored in maching learning s3 buckets.
From a user point of view this is just an API that does some prediction, you should focus on explaining what a prediction is, and what does it do, how should the user of this API use this prediction?
On top of that, if we decide in the future to change the technical implementation of these endpoints, this doc won't be accurate anymore.